main.js

```diff
@@ -0,0 +1,379 @@
+function readyPageFunctions() {
+    $(".red").on("click", playerColor);
+    $(".blue").on("click", playerColor);
+    $(".gold").on("click", playerColor);
+    $(".green").on("click", playerColor);
+}
+
+function addClickHandler() {
+    $(".col0").on('click', function(){
+        columnNumber = 0;
+        addToGridArray();
+    });
+
+    $(".col1").on('click', function(){
+        columnNumber = 1;
+        addToGridArray();
+    });
+
+    $(".col2").on('click', function(){
+        columnNumber = 2;
+        addToGridArray();
+    });
+
+    $(".col3").on('click', function(){
+        columnNumber = 3;
+        addToGridArray();
+    });
+
+    $(".col4").on('click', function(){
+        columnNumber = 4;
+        addToGridArray();
+    });
+
+    $(".col5").on('click', function(){
+        columnNumber = 5;
+        addToGridArray();
+    });
+
+    $(".col6").on('click', function(){
+        columnNumber = 6;
+        addToGridArray();
+    });
+
+    $(".reset").on('click', function() {
+        resetGame();
+    });
+
+    $(".toggleAI").on('click', function() {
+        toggleAI();
+    });
+
+    $(".togglePlayerNumber").on('click', function() {
+        togglePlayerMode();
+    });
+}
+
+var playerSwitch = 2; //variable that switches between players one, two and three
+var playerOneColor = null; // store player one's image
+var playerTwoColor = null; // store player two's image
+var playerThreeColor = null; // store player three's image
+var victoryTrigger = 0; //var that becomes 1 when game victory, disables certain interactive functions;
+var gameArray = [[null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null],
+    [null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null]];
+var columnNumber = null;
+var playerModeToggle = 1;
+
+function togglePlayerMode() {
+    $('.title').text("Third Player Summoned. Player One: Select Your Deity.")
+    $('.togglePlayerNumber').text("TWO PLAYER MODE");
+    playerModeToggle = 1 - playerModeToggle; //if playerModeToggle === 0; three player mode activates; else two player mode;
+    if (playerModeToggle === 1) {
+        $('.title').text("Third Player Sacrificed. Player One: Select Your Deity.");
+        $('.togglePlayerNumber').text("THREE PLAYER MODE");
+    }
+    toggleAICount = 0;
+}
+
+var toggleAISwitch = 1; //toggle that switches between player and AI
+var toggleAICount = 0;
+function toggleAI() { //toggles whether AI should be on/off
+    toggleAICount = 1 - toggleAICount;
+    $('.title').text("Choose your color and play with the Gods.");
+    if (toggleAICount === 0 ) {
+        $('.title').text("Player One: Choose Your Diety");
+    }
+    playerModeToggle = 1;
+    $('.togglePlayerNumber').text("THREE PLAYER MODE");
+};
+
+//Function to assign image class to players
+function playerColor() {
```

Comment:
There's lots of things talking about colors, but none of your game is based on colors. These variable names no doubt predate a shift in your theme, but make sure you update these sorts of things to keep inline with the game as it currently exists.